### ISSUE TITLE
Inconsistency between paper and code (Loss function in Starfysh with H&E integration)

### ISSUE THREAD

**Issue by GQ-ZHU** [azizilab/starfysh#64]
**Hello, thanks a lot for sharing this great work and the codebase!**

While reproducing the Starfysh model with H&E image integration, I noticed that the loss function implementation in the code seems inconsistent with the description in the paper.

### Paper description
In the paper (page 15-16, section:**Starfysh structured variational inference with histology integration.**), the objective is defined as:
 
<img width="565" height="395" alt="Image" src="https://github.com/user-attachments/assets/df531064-7b09-4b20-80e5-4fd26a0b631b" />

### Code implementation
In the current code (`starfysh.py`, function `AVAE_PoE.get_loss()`):

<img width="311" height="232" alt="Image" src="https://github.com/user-attachments/assets/63af48d1-76ac-4c26-9cbf-b946f5365fdb" />

<img width="437" height="80" alt="Image" src="https://github.com/user-attachments/assets/9cedb326-d5af-4975-b509-61b4e5a1fb43" />

### Observed difference
Paper: $L_{total} = L_{joint} + 5 * L_{marginal}$
Code: $L_{total} = 0.2 * L_{joint} + L_{marginal}$
So the relative weight between joint and marginal objectives seems reversed between paper and code.

**Would you be able to kindly clarify whether the discrepancy comes from the paper description or the code implementation, and which version should be considered the authoritative reference? Any clarification on this would be greatly appreciated.**

**Comment by YinuoJin** [comment]
Hi @GQ-ZHU thanks for using Starfysh 

Mathematically the $\alpha$ term is equivalent btw the "paper" and "code" version: $L_{tot} = L1 + \alpha*L2$ <==> $L_{tot} = 1/\alpha L1 + L2$. Multiplying the objective by a scalar doesn't affect the optimization as the gradient descent steps would scale accordingly. 

Here we wanted to emphasize more on the marginal loss in the PoE term. Hope it clarifies.

**Comment by GQ-ZHU** [comment]
Thank you for your clarification. I have considered that the two are equivalent (coefficients 0.2 for L1, 5 for L2), but then I thought that the values of L1 and L2 are different after all, and there might be a gap in order of magnitude. In conclusion, thank you very much for your reply!
